libs/blocks/media/media.js

```diff
@@ -35,6 +30,7 @@ export default function init(el) {
     decorateBlockBg(el, head);
     rows = tail;
   }
+  const blockType = el.classList.contains('merch') ? 'merch' : false;
```

Comment:
Nit: it feels a bit weird to have a type of `false`, maybe `null` or `undefined` might be better suited for this naming

Reply:
Good call, I changed last night w/ weary eyes. I agree. Thanks for the feedback on these